operator/v1/types_image_content_source_policy.go

```diff
@@ -0,0 +1,99 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageContentSourcePolicy holds cluster-wide information about how to handle registry mirror rules.
+// When multiple policies are defined, the outcome of the behavior is defined on each field.
+//
+// +kubebuilder:storageversion
```

Comment:
this can go. We don't use CRD generation of kubebuilder, with exception of schema generation.

Reply:
Fixed.